ci/jenkins/Jenkins_steps.groovy

```diff
@@ -515,6 +515,48 @@ def compile_windows_cpu() {
     }]
 }
 
+def compile_windows_cpu_mkldnn() {
+    return ['Build CPU MKLDNN windows':{
+      node(NODE_WINDOWS_CPU) {
+        ws('workspace/build-cpu-mkldnn') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            utils.init_git_win()
+            powershell 'py -3 ci/build_windows.py -f WIN_CPU_MKLDNN'
+            stash includes: 'windows_package.7z', name: 'windows_package_cpu_mkldnn'
+          }
+        }
+      }
+    }]
+}
+
+def compile_windows_cpu_mkldnn_mkl() {
+    return ['Build CPU MKLDNN MKL windows':{
+      node(NODE_WINDOWS_CPU) {
+        ws('workspace/build-cpu-mkldnn-mkl') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            utils.init_git_win()
+            powershell 'py -3 ci/build_windows.py -f WIN_CPU_MKLDNN_MKL'
+            stash includes: 'windows_package.7z', name: 'windows_package_cpu_mkldnn_mkl'
+          }
+        }
+      }
+    }]
+}
+
+def compile_windows_cpu_mkl() {
+    return ['Build CPU NOMKLDNN MKL windows':{
+      node(NODE_WINDOWS_CPU) {
+        ws('workspace/build-cpu-nomkldnn-mkl') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            utils.init_git_win()
+            powershell 'py -3 ci/build_windows.py -f WIN_CPU_NOMKLDNN_MKL'
```

Comment:
The build flavor(-f) should be `WIN_CPU_MKL` instead of `WIN_CPU_NOMKLDNN_MKL` since it has been modified from the latter in `ci/build_windows.py` (line 93).

Reply:
thank you for fix. change it now.